source/styles/settings.css

```diff
@@ -7,7 +7,8 @@
 
```

Comment:
Specify the context of fallback and the logic behind the formula in the comments